pkg/util/util.go

```diff
@@ -391,3 +391,17 @@ func EncodeURL(href string) (string, error) {
 	u.RawQuery = u.Query().Encode()
 	return u.RequestURI(), nil
 }
+
+func RemoveDuplicateStr(strSlice []string) []string {
+	allKeys := make(map[string]bool)
+	var list []string
+	for _, item := range strSlice {
+		if _, value := allKeys[item]; !value {
+			allKeys[item] = true
+			if len(item) > 0 {
```

Comment:
is there a need to TrimSpace the item?

Reply:
As we would have child names directly from `child.GetAllChildNamesS()`, we may not be needed TrimSpace the item.